modules/@apostrophecms/ui/ui/apos/components/AposCellDate.vue

```diff
@@ -22,24 +23,31 @@ export default {
   },
   methods: {
     formatDateColumn (value) {
-      const d = new Date(value);
-      if (Number.isNaN(d.getDate())) {
-        // We're not sure what this is, but it's not a date.
-        return value;
-      }
-      const month = d.getMonth();
-      const date = d.getDate();
-      const year = d.getFullYear();
-      let hour = d.getHours();
-      const period = hour > 11 ? 'pm' : 'am';
-      if (hour > 12) {
-        hour = hour - 12;
-      } else if (hour === 0) {
-        hour = 12;
-      }
-      const minute = d.getMinutes();
+      const format = this.$t('apostrophe:dayjsCellDateFormat');
+      // endsWith allows it to still match when the i18n show debug mode
+      // is in effect
+      if (format.endsWith('apostrophe')) {
+        const d = new Date(value);
+        if (Number.isNaN(d.getDate())) {
+          // We're not sure what this is, but it's not a date.
+          return value;
+        }
+        const month = d.getMonth();
+        const date = d.getDate();
+        const year = d.getFullYear();
+        let hour = d.getHours();
+        const period = hour > 11 ? 'pm' : 'am';
+        if (hour > 12) {
+          hour = hour - 12;
+        } else if (hour === 0) {
+          hour = 12;
+        }
+        const minute = d.getMinutes();
 
-      return `${toTwoChars(month)}/${toTwoChars(date)}/${toTwoChars(year)} at ${hour}:${minute} ${period}`;
+        return `${toTwoChars(month)}/${toTwoChars(date)}/${toTwoChars(year)} at ${hour}:${minute} ${period}`;
```

Comment:
Couldn't most of this be replaced with:
```
const d = new Date(value);
if (Number.isNaN(d.getDate())) {
  return value;
}
dayjs(d).format('MM/DD/YY at h:mm a');

```

Reply:
Probably, but again not new in the PR and I don't want to introduce work or potential bugs this close to shipping localiation, so I'm going to suggest you open a followup ticket for that.